### PR TITLE
[KEYCLOAK-8959] Add keycloak-test-helper into the BOMs

### PR DESCRIPTION
**Pull request by abstractj**

### CODE REVIEW

**Comment by abstractj** [comment]
@martin-kanis @stianst could you please review? I don't have the rights to assign, or add a reviewer.

**Review by stianst — APPROVED** [review]

**Comment by martin-kanis** [comment]
@abstractj Please add little note to README when and how to use this BOM.

**Review by martin-kanis — CHANGES_REQUESTED** [review]
@abstractj Please add little note to README when and how to use this BOM.

File: pom.xml [hunk 1]
```diff
@@ -71,7 +71,8 @@
     </dependencyManagement>
 
     <modules>
-	<module>adapter</module>
+	      <module>adapter</module>
```
Comment (martin-kanis) [inline pom.xml:74]: Different indentation :)

Comment (abstractj) [reply pom.xml:74]: @martin-kanis I believe the current indentation is broken. Take a look at https://github.com/redhat-developer/redhat-sso-boms/blob/7.3.x/pom.xml#L74. So the idea of this change is to fix it.

**Review by martin-kanis — APPROVED** [review]

**Pull request merged**